core/src/main/java/org/openstack4j/openstack/internal/BaseOpenStackService.java

```diff
@@ -1,14 +1,7 @@
 package org.openstack4j.openstack.internal;
 
-import static org.openstack4j.core.transport.ClientConstants.HEADER_USER_AGENT;
-import static org.openstack4j.core.transport.ClientConstants.USER_AGENT;
-
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.SortedSet;
-
+import com.google.common.base.Function;
```

Comment:
Actually don't use intellij anymore, but I think it was the "optimize imports" setting that needed to be deselected